### PR TITLE
Pillowbugs

### LINKED ISSUE #20171: Chillwax does not work on simple mobs
(WEB REPORT BY: repertor REMOTE: 172.93.109.202:7777)
# Revision
dcc4c15a29fe48e7d730fda0cd833d93054299f3

# Description
When injected, simple mobs are still able to attack
# Steps to Reproduce
inject a hostile mob with chillwax
# What you Expected
pacification of the mob
# What Actually Happened
continues to attack

### PR DESCRIPTION
**Pull request by MadmanMartian**
- Adds Pillowbugs, sprite and idea by Skowron.

 - - They don't cause any direct harm, but stop people from attacking one another if they're in view of them.

 - Simple animals can now be pacified, assuming they have the pacifiable flag

closes #20171 

NEEDS SPRITES FOR - Pillowbug in-hand

:cl:
 * rscadd: Something has begun to feed on the conflict within the asteroid

### CODE REVIEW

File: code/modules/mob/living/holders.dm [hunk 1]
```diff
@@ -187,3 +187,10 @@
 /obj/item/weapon/holder/animal/slime/attack_self(mob/user)
 	..()
 	unfreeze()
+
+/obj/item/weapon/holder/animal/pillow
+	name = "pillow holder"
+	desc = "Comforbable"
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/modules/mob/living/holders.dm]: :b:

File: __DEFINES/setup.dm [hunk 2]
```diff
@@ -577,6 +577,7 @@ var/list/global_mutations = list() // list of hidden mutation things
 #define CANKNOCKDOWN	2
 #define CANPARALYSE	4
 #define CANPUSH		8
+#define PACIFIABLE 16		//Can not be pacified
```
Comment (Exxion) [inline __DEFINES/setup.dm]: This comment is neither helpful nor correct

Comment (MadmanMartian) [reply __DEFINES/setup.dm]: dang, forgot to update it

File: code/modules/mob/living/holders.dm (hunk 1 above)
Comment (Shadowmech88) [reply code/modules/mob/living/holders.dm:193]: >comfortable pillbug
>comf-orb

This may have actually been intentional.

Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [reply code/modules/mob/living/holders.dm]: photoshop_2018-10-03_19-29-33

**Comment by Killette2** [comment]
nigga wtf is a pillowbug

**Comment by mikepmonson** [comment]
Could I hold onto one of these discreetly and prevent combat in the area? (That isn't gun related)

**Comment by DamianX** [comment]
something comfor:b:able

**Comment by MadmanMartian** [comment]
Free the borb

**Comment by MadmanMartian** [comment]
borb?

**Comment by MadmanMartian** [comment]
borb!?

**Pull request merged**